Microsoft.Toolkit.HighPerformance/Microsoft.Toolkit.HighPerformance.csproj

```diff
@@ -24,17 +24,19 @@
   </Description>
     <PackageTags>Windows;Community;Toolkit;WCT;UWP;core;standard;unsafe;span;memory;string;array;stream;buffer;extensions;helpers;parallel;performance</PackageTags>
   </PropertyGroup>
+
   <Choose>
-    <When Condition=" '$(TargetFramework)' == 'netstandard1.4' ">
+    <When Condition="'$(TargetFramework)' == 'netstandard1.4'">
+      <!--
+          .NET Standard 1.4 lacks the [Pure] attribute, the Rectangle primitive,
+          the Span<T> and Memory<T> types, the Vector<T> primitive and related APIs,
+          ValueTask and ValueTask<T>, the Parallel class and the Unsafe class.
+          We also need to reference the System.Runtime.CompilerServices.Unsafe package directly,
+          even though System.Memory references it already, as we need a more recent version than
+          the one bundled with it. This is so that we can use the Unsafe.Unbox<T> method,
+          which is used by the Box<T> type in the package.
+      -->
```

Comment:
I'm fine with the new blank lines around the `<Choose>` block and before the `<When>` blocks, and I can see how that can help improve readability. I don't like the arbitrary change of multiline comment style though, as well as the inverted order of defined constants and package references. Also the same remarks about comments I left above apply here as well, with the addition that comments were originally at the same level of the items they were referring to - why the change one level up instead?

Reply:
When a comment is common to a whole block of code and not just the following line, placing one level above gives clarity.

_Same remarks as above for the rest of the changes._